msg/versioned/BatteryStatus.msg

```diff
@@ -1,74 +1,77 @@
+# Battery status
+
 uint32 MESSAGE_VERSION = 0
 
-uint64 timestamp				# time since system start (microseconds)
-bool connected					# Whether or not a battery is connected, based on a voltage threshold
-float32 voltage_v				# Battery voltage in volts, 0 if unknown
-float32 current_a				# Battery current in amperes, -1 if unknown
-float32 current_average_a			# Battery current average in amperes (for FW average in level flight), -1 if unknown
-float32 discharged_mah				# Discharged amount in mAh, -1 if unknown
-float32 remaining				# From 1 to 0, -1 if unknown
-float32 scale					# Power scaling factor, >= 1, or -1 if unknown
-float32 time_remaining_s			# predicted time in seconds remaining until battery is empty under previous averaged load, NAN if unknown
-float32 temperature				# Temperature of the battery in degrees Celcius, NaN if unknown
-uint8 cell_count				# Number of cells, 0 if unknown
+uint64 timestamp # Time since system start [us]
+bool connected # Whether or not a battery is connected, based on a voltage threshold
+float32 voltage_v # Battery voltage [V] [@invalid 0]
+float32 current_a # Battery current [A] [@invalid -1]
+float32 current_average_a # Battery current average (for FW average in level flight) [A] [@invalid -1]
+float32 discharged_mah # Discharged amount [mAh] [@invalid -1]
+float32 remaining # Remaining capacity [@range 0,1] [@invalid -1]
+float32 scale # Power scaling factor [@range 1,] [@invalid -1]
```

Comment:
This case of `@range` is ugly. Should we allow @minValue, @maxValue for ranges that only define one bound?

Reply:
I don't see the issue. It's not ambiguous with the syntax and even though an infinite scale doesn't make sense, it would be strange to just write some arbitrary limit.

Reply:
Certainly not ambiguous. Not strange either - it is normal to have a range clamped at only one end for some things.